src/components/ui/Select/Select.tsx

```diff
@@ -87,31 +85,38 @@ const Select = ({
       {isOpen && (
         <div className={style.optionBox}>
           <ul className={style.options}>
-            {options?.map((el) => (
-              <li
-                key={el}
-                className={clsx(
-                  style.optionsItem,
-                  value && value === el && style.activeItem,
-                  type === 'filter' && style.filter,
-                )}
-                onClick={() => {
-                  handleSelectOption(el);
-                  if (type === 'filter' && value === el) {
-                    handleSelectOption('');
-                  }
-                }}
-              >
-                {el}
-                {type === 'filter' && value && value === el && (
-                  <Icon
-                    id="icon-close"
-                    size={28}
-                    stroke="var(--primary-dark)"
-                  />
-                )}
-              </li>
-            ))}
+            {options?.map((option) => {
+              const optionLabel =
+                typeof option === 'string' ? option : option.label;
+              const optionValue =
+                typeof option === 'string' ? option : option.value;
+
```

Comment:
I think better always pass object in options. If we don't need label just leave it empty or make the same like value. In this case we always know what type of data we have and pass